src/main/java/space/obminyashka/items_exchange/model/User.java

```diff
@@ -23,9 +23,11 @@ public class User extends BaseEntity implements UserDetails {
     @Column(unique = true)
     private String email;
     private Boolean online;
-
     private Boolean oauth2Login;
 
+    @Column(name = "validated_email")
+    private Boolean validatedEmail;
```

Comment:
While you're using Column here you can name the filed according to Java convention using "is" as a prefix